<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellSizeDialog</class>
 <widget class="QDialog" name="CellSizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>215</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>215</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Параметры ячейки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Введите количество входов и выходов элемента.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;В скобках указаны минимально и максимально &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;возможные значения.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inInfoLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="inInfoLabel">
       <property name="text">
        <string>Кол-во входов:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>14</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="requiredInCount">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="requiredInCountSB"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="outInfoLayout">
     <item>
      <widget class="QLabel" name="outInfoLabel">
       <property name="text">
        <string>Кол-во выходов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="requiredOutCount">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="requiredOutCountSB"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="isAutoCorrect">
     <property name="text">
      <string>разрешить автоисправление</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="errorMessageOutput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CellSizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CellSizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>requiredOutCount</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>requiredOutCountSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>requiredOutCountSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>requiredOutCount</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>requiredInCount</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>requiredInCountSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>requiredInCountSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>requiredInCount</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>requiredOutCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>requiredOutCountSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>requiredInCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>requiredInCountSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
